src/main/scala/interfaces/Fdi.scala

```diff
@@ -0,0 +1,406 @@
+package edu.berkeley.cs.ucie.digital.interfaces
+
+import chisel3._
+import chisel3.util._
+
+/** The flit-aware die-to-die interface (FDI), from the perspective of the
+  * protocol layer.
+  */
+class Fdi(width: Int, dllpWidth: Int, sbWidth: Int) extends Bundle {
+
+  /** Protocol layer to Adapter data.
+    *
+    * Encompasses `lp_irdy`, `lp_valid`, and `pl_trdy` from the UCIe
+    * specification.
+    */
+  val lpData = Decoupled3(Vec(width, UInt(8.W)))
```

Comment:
While this is how the spec defines this wire, is this how we want to define it from Chisel? (this applies to both directions, and the data on RDI as well)

The problem is that when this elaborates to verilog (under the current elaboration pipeline), it will generate `width` individual wires of `[7:0]` (i.e. `data_0[7:0]`, `data_1[7:0]`, etc.). We're already not compatible with the spec's signal names and would need a separate type if we do ever want to expose 1:1 compatible names (though I don't know if it is even possible to generate a 2D array). However for our own use (from within Chisel), this also means we have the option of just replacing it with a `Bits((8 * width).W)`. The main parts this would affect is probably only testbenches interacting with the Chisel (whether it is SV testbenches that need to read/drive this interface or just looking at waveforms from a purely Chisel-generated testbench).

Reply:
changed this to be `Bits`. I don't think this is likely, but if the bit/byte ordering becomes confusing, we may need to revisit this.